src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.scss

```diff
@@ -5,15 +5,31 @@
     margin: 0 0 30px 0;
   }
 
+  .form-group {
+    border-left: 4px solid white;
+    background-color: $color-password-toggle-bg;
+    height: auto;
+    margin-bottom: 2rem;
+  }
+
+  #username-div.color,
+  #password-div.color {
+    border-left: 4px solid $color-accent;
+  }
```

Comment:
Just found out about this. You can use :focus-within to check if any child component is focused.
With this you don't need to keep changing classes in the html.

.invalid-feedback is not working well with this type of forms, add this style to fix it.

```suggestion
    &:focus-within {
      border-left: 4px solid $color-accent;
    }
  }

  .invalid-feedback {
    display: block;
  }
```
I would suggest adding more left padding to the error messages:
![image](https://user-images.githubusercontent.com/399326/81329221-bfbf5f80-908d-11ea-9308-dd07dfadda1f.png)